content/post/articles/how-to-build-an-oss-project.md

```diff
@@ -0,0 +1,139 @@
+
+It is important to set a **daily, weekly, or monthly catch-up** on which you can go through the issues you have created and discuss the evolution of the project. For **jobsika**, we have set a weekly meeting that was sometimes canceled as some of us were busy during specific periods.
+
+Be mindful of other people’s time. It is acceptable for your collaborators to show up at a meeting with no work to report. They were probably busy or not inspired. What is essential is not to lie and be transparent about what you have accomplished. In some cases, a contributor won’t be productive for about two months or not be very present. Their motivation to work on the project might be in question, which should be discussed with them. They might need to quit the project for a while.
+
+Track issues using [GitHub projects][1]. You can also use a kanban board during your daily or weekly catch-ups. We use GitHub projects to track [the roadmap][2] at **osscameroon**. So feel free to check it out for inspiration.
+
+**Keep the communication flowing:**
+
+Once you have formed a team to work on your project, it can be helpful to create a chat group for you to share information, ask for help, and discuss everything related to the project.
+
+For **jobsika** and other projects at osscameroon, we created telegram groups. Using telegram group helps us kick off instant voice chat, which is pretty neat.
+
+**Setting realistic milestones:**
+
+As we discussed already, keeping your team motivated will be challenging and making sure you see important results quickly will be critical.
+
+Developers tend to be very optimistic about how long a task will take. For example, let’s say a developer picks up the implementation of a search bar. In their mind they will think it is a 1h work. If you give them one week to work on such a task, they might wait for the last hour before the meeting to start working on the actual ticket. And might realize that they did not consider the filters in the search bar. This means you need more time to deliver that essential feature.
+
+We don’t have an appropriate solution for that yet, but something that can help speed things up is to set a milestone that can match the 4-5 hours a week each team member can provide. So arbitrarily, we could say that for a single developer working 5 hours a week implementing a well-scoped (removed of unnecessary features) home page can be achieved in a month.
+
+You could then expect the home page to be ready in a month, but as we often observe 99% of the time, this won’t be true. But at least you might have something close to a fully functional home page.
+
+#### The Architecture
+
+Regarding the architecture of your project, the keyword here is **simplicity**. As developers, we often like to work on complicated systems, implement challenging algorithms and use the new fancy DevOps practices or tools (Kubernetes) we saw on Twitter. These are all noble things to thrive for, but they come with a cost.
```

Comment:
```suggestion
Regarding the architecture of your project, the keyword here is **simplicity**. As developers, we often like to work on complicated systems, implement challenging algorithms and use the new fancy DevOps practices or tool (Kubernetes) we saw on Twitter. These are all noble things to thrive for, but they come with a cost.
```

If you want to bash on Kubernetes, you should use the singular.

Reply:
Thanks, Kubernetes here is an example of tools we tend to overuse. That's why I would prefer to keep the plural.